docs/setup/install-with-keink.md

```diff
@@ -0,0 +1,121 @@
+# Deploying KubeEdge with Keink
+
+keink(represent for [KubeEdge](https://github.com/kubeedge/kubeedge) IN [kind](https://github.com/kubernetes-sigs/kind)) is a tool for running local KubeEdge clusters using Docker container "nodes".It allows for the easy deployment of a KubeEdge cluster on a single machine. This method is only used for trial kubeedge, please do not use it in a production environment.
+
+## Environment Preparation
+
+### KubeEdge Codes
+
+You need to download the KubeEdge codes locally. It's recommended to place it in the `$GOPATH` directory, as keink can automatically detect the KubeEdge source code in this directory. Alternatively, you can manually enter the code location.
```

Comment:
I think we should draft a release for keink, users don't need clone keink code and build themselves. @fisherxu @wbc6080

Reply:
+1